manager/notifier.go

```diff
@@ -87,6 +87,7 @@ var (
 	smtpSender             = flag.String("notification.smtp.sender", "alertmanager@example.org", "Sender email address to use in email notifications.")
 	hipchatURL             = flag.String("notification.hipchat.url", "https://api.hipchat.com/v2", "HipChat API V2 URL.")
 	flowdockURL            = flag.String("notification.flowdock.url", "https://api.flowdock.com/v1/messages/team_inbox", "Flowdock API V1 URL.")
+	slackConnectTimeout    = flag.Int("notification.slack.timeout", 10, "HTTP timeout to talk to slack (in seconds).")
```

Comment:
Oh, actually can't merge it: "This branch has conflicts that must be resolved". Please rebase :)